lib/Client.js

```diff
@@ -1779,10 +1779,16 @@ class Client extends EventEmitter {
     * Modify a guild's widget
     * @arg {String} guildID The ID of the guild
     * @arg {Object} options The widget object to modify (https://discord.com/developers/docs/resources/guild#modify-guild-widget)
-    * @arg {String} [options.reason] The reason to be displayed in audit logs
+    * @arg {Boolean} [options.enabled] Whether the guild widget is enabled
+    * @arg {String?} [options.channelID] The channel ID for the guild widget
+    * @arg {String?} [options.reason] The reason to be displayed in audit logs
     * @returns {Promise<Object>} A guild widget object
     */
     editGuildWidget(guildID, options) {
+        if(options.channelID) {
```

Comment:
Check specifically if not undefined
```suggestion
        if(options.channelID !== undefined) {
```